# This represents the the forces the bluefin submarine's engine give at some point.
Header header


float32 force		# the force the engine apllies.
float32 u_angle		# the up-down angle of the propeller.
float32 l_angle		# the left-right angle of the propeller.
